packages-backend/loggers/src/middlewares/create-access-logger.ts

```diff
@@ -19,7 +18,8 @@ const createAccessLoggerMiddleware = (options: TAccessLoggerOptions = {}) => {
     meta: true,
     expressFormat: true, // Use default morgan access log formatting
     colorize: !options.json,
-    skip: (req) => !shouldLog || ignoreUrls.includes(req.originalUrl),
+    skip: (req) =>
+      Boolean(options.silent) || ignoreUrls.includes(req.originalUrl),
```

Comment:
Almost feel we could achieve the same with instead of having ignoreUrls having the skip function as an option. So it would be like

```js
skip (...args) => Boolean(options.silent) || options.skip(...args)
```

I guess that would mean some refactoring in our apps though.

Reply:
Yeah, I think passing a list of URLs is less work and more straight-forward though. After all the intent with the package is to have some sort of pre-configured loggers. 🤗